tracer/src/Datadog.Trace/Configuration/ConfigurationKeys.Exporter.cs

```diff
@@ -94,5 +94,12 @@ internal static partial class ConfigurationKeys
         /// </summary>
         /// <seealso cref="ExporterSettings.MetricsUnixDomainSocketPath"/>
         public const string MetricsUnixDomainSocketPath = "DD_DOGSTATSD_SOCKET";
+
+        /// <summary>
+        /// Configuration key for the Metrics location where the Tracer can send traces.
```

Comment:
Just kinda got confused with the comment here, maybe it could be changed like the below as it is in the port config:
> Configuration key for the location where the Tracer can send DogStatsD metrics.